info_extraction/requirements.txt

```diff
@@ -0,0 +1,3 @@
+beautifulsoup4==4.6.0
+markdown==1.7
```

Comment:
Same for markdown, do you need this version of markdown specifically?

Reply:
I think this is what I saw was used from the code that we re-used. I believe there is a citation / web link to it there too.